frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -315,7 +315,49 @@ common_settings.document = {
                     end,
                 },
 
-            }
+            }		
+        },        
+	{
+            text = _("Action on highlight"),
```

Comment:
There are too many spaces here.

```suggestion
        text = _("Highlight action"),
```